storage/client_test.go

```diff
@@ -795,6 +795,62 @@ func TestOpenReaderEmulated(t *testing.T) {
 	})
 }
 
+func TestOpenReaderEmulated_Metadata(t *testing.T) {
+	transportClientTest(skipHTTP("metadata on read not supported by JSON api"), t, func(t *testing.T, ctx context.Context, project, bucket string, client storageClient) {
```

Comment:
I believe the HTTP client should use XML by default, does the test not pass with a default HTTP client?

Reply:
right you are.